code/game/gamemodes/game_mode.dm

```diff
@@ -230,7 +230,11 @@ var/global/list/additional_antag_types = list()
 	refresh_event_modifiers()
 
 	spawn (ROUNDSTART_LOGOUT_REPORT_TIME)
-		display_roundstart_logout_report()
+		var/logout_report = get_logout_report()
+		for(var/s in staff)
+			var/client/C = s
+			if(check_rights(R_MOD|R_ADMIN,0,C))
+				to_chat(M, logout_report)

```

Comment:
This is still referencing M which does not exist. idk if to_chat can send stuff to client? But client probably has reference to a mob somewhere